arkaisho/my_eyes_app
# Feature/authentication store

What i did:
- Created authentication store and api functions
- Created product pages
   - Product list page
   - Product creation page
   - Product edition page
How to test:
- Run the app
- Be sure that all the authentication functions are working on the api
- Test the authentication features (login, user creation and password recover)
- See if the product pages looks just like the mockup
